### DISCUSSION TITLE
新版Squirrel 0.15在Mac OS 10.10上行为怪异

### DISCUSSION THREAD

**Discussion by easior** [rime/home#700] [Q&A]
檢測到新版後即刻更新，可是更新後行爲很奇怪。
原先默認豎排的選單變成了橫排，
而且選單數量超過後不能翻頁，例如後面的截圖，
這是否說明新版不適用於Mac OS 10.10，或者有特別的設置？
![Capture d’écran 2021-02-11 à 18 29 57](https://user-images.githubusercontent.com/613830/107625682-a6d88000-6c97-11eb-95b7-03ec9889727a.png)

**Comment by lotem** [comment]
同 https://github.com/rime/squirrel/issues/513

**Comment by easior** [comment]
今天测试了一下15.2，基本上翻页、横排等问题已解决；
但很奇怪，不知道怎么设置选单竖排？例如设置
`"style/horizontal": false`
的结果是选单只显示一个字，且无法翻页（见附图）。
还有很奇怪，我将配置改为
```
patch:
     style: 
       horizontal: false
```
输入法直接停止工作，不知道这是配置格式的问题还是其他？

PS.
本来想将反馈写到 rime/squirrel#513 中，但看起来它以被关闭。
![Capture d’écran 2021-02-14 à 09 27 51](https://user-images.githubusercontent.com/613830/107866356-e1663680-6eaa-11eb-8d85-99a0fd9c64fc.png)

**Reply by lotem** [reply]
如果問題沒有解決可以重新打開或直接繼續跟帖。

**Reply by LEOYoon-Tsaw** [reply]
要用
```
  "style/candidate_list_layout": linear # linear     | stacked
  "style/text_orientation": vertical #    horizontal | vertical
```
設置

**Reply by lotem** [reply]
設置 `style/horizontal` 也應該可以。

**Reply by easior** [reply]
@LEOYoon-Tsaw 使用该配置的效果请看截图
![Capture d’écran 2021-02-14 à 17 35 13](https://user-images.githubusercontent.com/613830/107873151-2b6f0c80-6eeb-11eb-8732-d06c1e267127.png)

**Reply by easior** [reply]
@lotem 使用原先的配置，竖排只有个选项

**Reply by lotem** [reply]
最好移步 https://github.com/rime/squirrel/issues 反饋問題。發到討論區不一定會持續跟進。
